lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -624,6 +633,33 @@ export async function addReviewers(
   }
 }
 
+async function retry<T extends (...arg0: any[]) => any>(
```

Comment:
```suggestion
async function retry<T extends (...arg0: any[]) => Promise<any>>(
```

why not enforce promise return type?

Reply:
My initial intention was to make the function more generic so it would work also with function that to not return a promise however in this context its a good idea to have a promise return type.